src/inmanta/deploy_project.py

```diff
@@ -1,506 +0,0 @@
-"""
-    Copyright 2018 Inmanta
-
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-        http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
-
-    Contact: code@inmanta.com
-"""
-
-import argparse
-import logging
-import os
-import socket
-import subprocess
-import sys
-import time
-from typing import Optional
-
-from inmanta import config, const, module, postgresproc, protocol
```

Comment:
I suspect `inmanta.postgresproc` can go as well

Reply:
I see. Only the typing used there. And it's even an incorrect type.